src/store/actions/resource.js

```diff
@@ -543,3 +543,13 @@ export const uploadResourceThumbnailAction = (formData) => async (dispatch) => {
     // throw new Error(e);
   }
 };
+
+export const activeShareActivity = async (actvityId) => {
```

Comment:
I think function name should be shareActivity
Also it seems like result is redundant

Reply:
done